src/PdfReader/lib.ts

```diff
@@ -33,6 +33,16 @@ export const loadResource = async (
   return array;
 };
 
+/**
+ * Fetches a resource url as text
+ */
+export async function getContentDefault(
+  resourceUrl: string,
+  proxyUrl?: string
+): Promise<Uint8Array> {
+  return loadResource(resourceUrl, proxyUrl);
+}
+
```

Comment:
My last thought is - if this just calls loadResource, should we just use that function directly and rename it `getContentDefault` or `fetchAsUint8Array`?